gui/src/main/java/org/jboss/as/console/client/shared/subsys/elytron/ui/factory/ConfigurableHttpServerMechanismFactoryFilterEditor.java

```diff
@@ -131,6 +131,8 @@ public String getValue(ModelNode property) {
                     .setSecurityContext(securityContext)
                     .build();
             addFormAssets.getForm().setFields(patternFilter, enabling);
```

Comment:
This is wrong in the current codebase, the manual add of the attributes, duplicates the enabling attribute.